packages/material-ui/src/Portal/Portal.test.js

```diff
@@ -86,6 +87,14 @@ describe('<Portal />', () => {
         return;
       }
 
+      beforeEach(() => {
+        consoleErrorMock.spy();
+      });
+
+      afterEach(() => {
+        consoleErrorMock.reset();
```

Comment:
Just curious what are these for?

Reply:
It uses the server side API but it also has windows defined, the useLayoutEffect / useEffect fork logic doesn't apply.

Reply:
The logic happens to use `useLayoutEffect` something React warn against in the server. I couldn't find a better alternative.